src/blockdata/constants.rs

```diff
@@ -84,7 +84,7 @@ fn bitcoin_genesis_tx() -> Transaction {
 
     // Outputs
     let out_script = script::Builder::new()
-        .push_slice(&Vec::<u8>::from_hex("04678afdb0fe5548271967f1a67130b7105cd6a828e03909a67962e0ea1f61deb649f6bc3f4cef38c4f35504e51ec112de5c384df7ba0b8d578a4c702b6bf11d5f").unwrap())
```

Comment:
Ah that's really cool :) I guess `Vec<u8>` is the only vec `FromHex` is defined on?

Reply:
It wouldn't make much sense on any other Vec :)  (without getting into endianness lol )